templates/taxi/driver_list.html

```diff
@@ -6,8 +6,13 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if user.id == driver.id %}
```

Comment:
The condition `user.id == driver.id` is used to check if the current user is the same as the driver in the list. Ensure that the `user` object is available in the template context and that both `user.id` and `driver.id` are correctly set.